packages/manager/apps/pci-load-balancer/src/hooks/useTranslatedLinkReference.ts

```diff
@@ -23,9 +26,10 @@ export const useTranslatedLinkReference = () => {
           anchor.setAttribute('data-handled', 'true');
           if (trackOn === 'click') {
             anchor.addEventListener('click', () => {
-              tracking.trackClick({
-                name: trackName,
-                type: trackType || 'action',
+              trackClick({
+                actions: [trackName],
+                actionType:
+                  (trackType as TrackingClickParams['actionType']) || 'action',
```

Comment:
What happens if the actionType isn't a valid param ? Could we add a predicate to verify the cast ?

Reply:
If trackType is not correct it will still take it as a valid 'actionType'
I wish I could easily but tracking param is define in useOvhTracking :
```typescript
export type TrackingClickParams = {
  location?: PageLocation;
  buttonType?: ButtonType;
  actions?: string[];
  actionType?: 'action' | 'navigation' | 'download' | 'exit';
};
```
The only thing I could see doing is add a 'isActionType' function to check is the string is in the list of possible values. It would be better to add it in the useOvhTracking file directly but I wanted to avoid touching 'manager-react-shell-client'.

I don't know if there is a better solution

Reply:
I think the isActionType solution is the best, but it's up to you if it is worth it to do the change

@fredericvilcot what do you think ?

Reply:
Yes, please add a type guard here to safely prevent from unexpected errors, because type cast only shunts `ts` engine and makes the code weaker.